src/Middleware/StaticFiles/src/DirectoryBrowserExtensions.cs

```diff
@@ -17,6 +18,8 @@ public static class DirectoryBrowserExtensions
     /// </summary>
     /// <param name="app"></param>
     /// <returns></returns>
+    /// <remarks>This defaults to browsing files from the path specified in <see cref="IWebHostEnvironment.WebRootPath"/>
+    /// which defaults to the 'wwwroot' subfolder.</remarks>
```

Comment:
Repeat for all: multiline XML elements should have element start/end on separate lines.